tests/storage/conftest.py

```diff
@@ -190,6 +192,15 @@ def skip_test_if_no_hpp_sc(cluster_storage_classes):
         pytest.skip(f"This test runs only on one of the hpp storage classes: {HPP_STORAGE_CLASSES}")
 
 
+@pytest.fixture(scope="session")
+def xfail_test_if_gcp_sc(cluster_storage_classes):
+    existing_gcp_sc = [sc.name for sc in cluster_storage_classes if sc.name in StorageClassNames.GCP]
+    if existing_gcp_sc:
+        pytest.xfail(
+            reason=f"This test is X failed due to non supporting GCP storage class dv less than 4Gi'{existing_gcp_sc}'"
+        )
```

Comment:
If my cluster has GCP and OCS, and I'm running the test on OCS, this would xfail anyway?